script/build

```diff
@@ -9,6 +9,7 @@ cd "$(dirname "$0")/.."
 mkdir -p ./site/public
 
 if [ -n "$CF_KV_TOKEN" ]; then
+  sleep 70 # Wait for KV writes to reach all nodes
```

Comment:
check the env variable if we got triggered or if it's based on a PR and only sleep on trigger

Reply:
Also 61 should be enough.